qiskit_ibm_runtime/accounts/management.py

```diff
@@ -67,6 +68,7 @@ def save(
                 channel=channel,
                 proxies=proxies,
                 verify=verify,
+                channel_strategy=channel_strategy,
```

Comment:
With this implementation, the `channel_strategy` will be saved only if `overwrite==True`. Is that the preferred behaviour?

Reply:
Yes, if the other credentials are the same, then the user must set `overwrite==True`